test/descriptive.test.js

```diff
@@ -30,6 +31,26 @@ describe('Descriptive Statistics', () => {
     testUndefinedWithNullable(max);
   });
 
+  test('Sum', () => {
+    expect(sum([])).toBe(0);
```

Comment:
Should be `undefined`.
```suggestion
    expect(sum([])).toBeUndefined();
```